fragment.go

```diff
@@ -293,11 +293,13 @@ func (f *Fragment) close() error {
 	// Flush cache if closing gracefully.
 	if err := f.flushCache(); err != nil {
 		f.logger().Printf("fragment: error flushing cache on close: err=%s, path=%s", err, f.path)
+		return err
```

Comment:
Hi Travis I spoke with @benbjohnson handling fragment `Close()` and he said he couldn't think of any reason why it shouldn't be handling the errors.  While I agree it probably does not make much difference during server shutdown, I think we should handle them during delete operations.    Unless you think otherwise I'll file tickets to handle the other `Close()` function errors.